.github/workflows/release-artifacts.yml

```diff
@@ -0,0 +1,17 @@
+name: Tag and Release
+
+on:
```

Comment:
Shouldn't this one also only run when the tests pass?

Reply:
This is true. We're pursuing to automate this release part and not tie it to a release we manually create. I'll update this workflow to reflect it.

Reply:
This has been addressed.